src/adapters/eplanning.js

```diff
@@ -0,0 +1,53 @@
+var bidfactory = require('src/bidfactory.js');
+var bidmanager = require('src/bidmanager.js');
+var adloader = require('src/adloader.js');
+
+var EPlanningAdapter = function EPlanningAdapter() {
+
+  var ISV_DEFAULT = "aklc.img.e-planning.net";
+
+  window.pbjs = window.pbjs || {};
+  window.pbjs.processEPlanningResponse = function(response) {
+    var bids, bidObject, i;
+    if (response) {
+      bids = response.bids;
+      for (i = 0; i < bids.length; i++) {
+        if (bids[i].ad) {
+          bidObject = getBidObject(bids[i]);
+          bidmanager.addBidResponse(bids[i].placementCode, bidObject);
+        } else {
+          bidObject = bidfactory.createBid(2);
+          bidObject.bidderCode = 'eplanning';
+          bidmanager.addBidResponse(bids[i].placementCode, bidObject);
+        }
+      }
+    }
+  };
+
+  function getBidObject(bid) {
+    var bidObject = bidfactory.createBid(1), i;
+    bidObject.bidderCode = 'eplanning';
+    for (i in bid.ad) {
+      if (bid.ad.hasOwnProperty(i)) {
+        bidObject[i] = bid.ad[i];
+      }
+    }
+    return bidObject;
+  }
+
+  function _callBids(params) {
+    var scriptURL = "http://" + ((params && params.bids && params.bids.length && params.bids[0].params && params.bids[0].params.isv) ? params.bids[0].params.isv : ISV_DEFAULT) + "/layers/hbpb.js";
+    adloader.loadScript(scriptURL, function() {
+      if (window.hbpb) {
+        window.hbpb.call(params);
+      }
```

Comment:
@mkendall07 or @protonate: How do you feel about prebid referencing global variables from other scripts?

This is obviously a really simple code block, but... I'm nervous about the coupling it creates. For example, if the loaded script set `window.hpbp = 5`, then this block would throw an error... and it wouldn't really be clear to me which script was "at fault" for it.

@sebaperez Could you give me some background on what this is trying to accomplish? There might be some way to accomplish the same goal without introducing cross-script state.